### ISSUE TITLE
no such table: json_each

### ISSUE THREAD

**Issue by prawnsalad** [mattn/go-sqlite3#954]
From CLI:
~~~
sqlite> select * from json_each('[5,6,7]');
0|5|integer|5|1||$[0]|$
1|6|integer|6|2||$[1]|$
2|7|integer|7|3||$[2]|$


$ sqlite3 --version
3.28.0 2019-04-15 14:49:49 378230ae7f4b721c8b8d83c8ceb891449685cd23b1702a57841f1be40b5daapl
~~~

From go-sqlite3:
~~~
rs, err := db.Exec("select * from json_each('[5,6,7]')")
if err != nil {
	log.Fatal("Error with json_each:", err.Error())
}

> 2021/07/12 16:06:21 Error with json_each:no such table: json_each
~~~

I would have expected the same result

**Comment by otoolep** [comment]
There is a lot of setup you're not showing. Maybe your Go code is buggy and you're not connecting to the database to you think you are.

**Comment by prawnsalad** [comment]
@otoolep the query doesn't reference anything that was previously setup  - it could be using `:memory:` or an empty database with 0 queries run before it.

~~~
$ sqlite3 ./none-existing-db.sqlite
SQLite version 3.28.0 2019-04-15 14:49:49
Enter ".help" for usage hints.
sqlite> select * from json_each('[5,6,7]');
0|5|integer|5|1||$[0]|$
1|6|integer|6|2||$[1]|$
2|7|integer|7|3||$[2]|$
~~~

~~~
db, err := sql.Open("sqlite3", "./foo.db")
rs, err := db.Exec("select * from json_each('[5,6,7]')")
if err != nil {
	log.Fatal("Error with json_each:", err.Error())
}
~~~

**Comment by prawnsalad** [comment]
Ahh.. I've just realised the extension list in the readme are the *available* ones during build, not included by default. `go run -tags sqlite_json ...` got it working as expected!

**Comment by chriseaton** [comment]
Thanks @prawnsalad i was stuck with the `no such table: json_each` error - the `go run -tags sqlite_json` solved it for me as well using v1.14.15
